salty/Main.hs

```diff
@@ -51,6 +51,9 @@ main  =
 
      (fsm, cont) <- genFSM opts input
 
+     unless (and (checkHasSuccessors fsm)) $
```

Comment:
```suggestion
     unless (all checkHasSuccessors fsm) $
```

Reply:
Whoops, nevermind -- for some reason I read this as mapping `checkHasSucessors` over `fsm`.